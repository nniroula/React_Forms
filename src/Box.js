import React from 'react';

// Box- this component should display a div with a background color, width and height based on the props 
// passed to it.

const Box = () => {
    return (
        <div>
            <p>Sytling for props goes here</p>
        </div>
    )
}

export default Box;