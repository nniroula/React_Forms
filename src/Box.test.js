import { render } from '@testing-library/react';
import Box from './Box';

test('renders without crashing', () => {
  render(<Box />);
});

test('it matches snapshot', () => {
    const {asFragment} = render(<Box />);
    expect(asFragment()).toMatchSnapshot()
})

// specialized tests, queries tests

test('playing with queries', () => {
    const { getByText } = render(<Box />);
    console.log(getByText('Sytling for props goes here'))
})