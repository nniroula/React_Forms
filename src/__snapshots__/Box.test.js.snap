// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <p>
      Sytling for props goes here
    </p>
  </div>
</DocumentFragment>
`;
